src/main/scala/firrtl/backends/verilog/VerilogEmitter.scala

```diff
@@ -57,6 +57,13 @@ object VerilogEmitter {
   private def precedenceGt(op1: PrimOp, op2: PrimOp): Boolean = {
     precedenceMap(op1) < precedenceMap(op2)
   }
+
+  /** Identifies PrimOps that never need parentheses
+    *
+    * These PrimOps emit either {..., a0, ...} or a0 so they never need parentheses
+    */
+  private val neverParens: PrimOp => Boolean =
+    Set(Shl, Cat, Cvt, AsUInt, AsSInt, AsClock, AsAsyncReset)
```

Comment:
What about adding `Pad`? I doubt this case gets hit in any code given the current inlining status, but it looks like it might be possible to get `({a0})` if you got the right tree in the emitter.

Reply:
Good call